### PR TITLE
Adds a new race, Saps

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
add: You can now play as a Sap. Just make sure to dress well as befitting your sophistication.
/:cl:

~~Totally not kinda stolen from Pyre!~~

Saps are posh treant-folk similar to podpeople. They are by nature vain divas, and are so obsessed with their self-image that they begin to starve if they're humiliated by wearing clothing that isn't as fancy as themselves.

Stats:
* Slower movement than default humans (around dufflebag speed by default; this is the big one)
* 60% less shock damage due to their planty flesh
* 10% less brute damage due to bark
* 10% more burn damage due to being plants
* Heat up 50% faster due to the above
* Dislikes meat and dairy, likes fruit, veggies, and grains
* Uses E-Z Nutrient for blood
* Heal from Robust Harvest and Left 4 Zed (the latter causes radiation!)
* Take high toxin damage from Plant-B-Gone
* Very slight nourishment from water
* Uses canopies instead of hairstyles

Saps' vanity is implemented in the form of fanciness. Every tick, saps will calculate their fanciness depending on the clothes they're wearing, with the default value being 50%. Below 50% fanciness, saps slowly lose hunger, and above 50% fanciness, they will be kept satiated.

Antagonists have a flat +20% fanciness due to their antagonistic nature. Clowns have +50% both because they are shameless and to counteract the high ugliness of their gear.

Clothing fits into six categories for fanciness:
* **Distasteful:** -10% fanciness per item, i.e. clown suits, chicken costumes, and bone armor
* **Ugly:** -5% fanciness per item, i.e. space suits, helmets, and hard hats
* **Neutral:** No change; most clothing is in this category
* **Fancy:** +5% fanciness per item, i.e. head jumpsuits, fancy glasses
* **Lavish:** +10% fanciness per item, i.e. captain parade gear
* **Chic:** +15% fanciness per item, i.e. monocles and bowler hats

Fanciness can be viewed by checking an alert in the top-right corner of your screen that shows how each article of clothing affects your fanciness.

The current two canopies are Oakley Traditional and Springston's Blooms, visible below.

All suggestions, feedback, and ideas are encouraged. I want this to be a roundstart race due to their uniqueness and inferiority to humans, so if there's some sort of condition that needs to be met to make that happen I can do my best to do it.

Balance as always is subject to change.

Pictures (currently they still use podman sprites):

*To-Do*
- [x] Random names
- [x] Reagent interactions
- [x] More canopies
- [x] Maintainer/player feedback
- [x] Possible: bark styles replace mutcolor, canopies use instead?

### CODE REVIEW

**Comment by Slignerd** [comment]
Post pics

**Comment by Xhuis** [comment]
They currently just use pod sprites, but I'll get some screenshots and post them in the op.

**Comment by The-clown-known-as-Pepe** [comment]
Vulnerable to plant-b-gone?
Turns corpse to charcoal when burned?

**Comment by Xhuis** [comment]
They'll have different interactions with the fertilizers/plant-killer; them breaking down into charcoal is probably doable but I'm unsure how good of an idea it is balancewise

**Comment by lzimann** [comment]
Man, this whole thing is quite bad

File: code/modules/mob/living/carbon/human/species_types/sap.dm [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+	siemens_coeff = 0.4 //They're plants and are less vulnerable to shocks
+	exotic_blood = "ez_nut"
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/plant
+	disliked_food = MEAT | DAIRY
+	liked_food = VEGETABLES | FRUIT | GRAIN
+	var/fanciness = 0 //Percentage of fanciness, determined by clothing worn. Below 50% fanciness you begin to starve, and above it you are nourished!
+	var/list/fancy_clothing_info //Used so saps can track their fanciness in an alert
+
+
+	var/static/list/despicable_clothing_typecache = typecacheof(list(\
```
Comment (lzimann) [inline code/modules/mob/living/carbon/human/species_types/sap.dm]: All these lists are quite awful, not worth having 5 typechaches for your snowflake "fancy"(what the fuck is even this or why does it exist) thing

Comment (Xhuis) [reply code/modules/mob/living/carbon/human/species_types/sap.dm]: It exists to give the race an interesting gimmick like plasmamen have; typecaches are only one way of doing it and I could also add a variable to `/obj/item/clothing` that serves the same purpose if it's that bad

**Comment by Xhuis** [comment]
I've been thinking about antagonists just getting a small bonus to the fanciness so that even though they're more concerned with their objectives their vanity still has an impact.

Though fanciness is their chief gimmick, it doesn't make or break the race; you can freely run around with a chicken suit, you'll just need to eat every once in a while since all high fanciness does is keep you satiated; at worst you move more slowly and have some small changes to your base protection values.

**Comment by vuonojenmustaturska** [comment]
why not make fanciness a clothing var?

**Comment by uraniummeltdown** [comment]
Just seems like a discount podman, if you want to play pod-dudes just get someone to plant clone you roundstart
Not well thought out imo

**Comment by jughu** [comment]
go back to para reee

**Comment by KorPhaeron** [comment]
Why?

**Comment by Slignerd** [comment]
@jughu "Go back to Para" would only be applicable if this were an actual Paradise race.

**Comment by Xhuis** [comment]
@uraniummeltdown I'd say that podman is more discount Sap; even though they share sprites currently, if the PR is given a fighting chance I'd like to give them customization options and make them distinct.

@KorPhaeron I think it's an interesting idea with good RP potential due to its implementation (saps are vain) and the idea has been floating around in my head for a while. It also has more differences than, say, lizards, so they're not human 2.0 to play as.

**Comment by rustledjimm** [comment]
I don't see what's wrong with this. It's different from humans so it isn't merely a sprite change like catpeople. It's balanced weaker than humans since it requires to maintain something to not starve not even including that the AI can just kill it when it wants. Seems like it would be a good addition given time, it's own sprites and some fleshing out.

**Comment by ExcessiveUseOfCobblestone** [comment]
They really just look like shitty variants of plant people.

Also

>Every tick, saps will calculate their fanciness depending on the clothes they're wearing, with the default value being 50%. Below 50% fanciness, saps slowly lose hunger, and above 50% fanciness, they will be kept satiated.

Wow.

**Comment by BeeSting12** [comment]
More fetish content

**Comment by Xhuis** [comment]
@ExcessiveUseOfCobblestone If I decide the PR has okay enough reception, sprites will be made to make them different from plants, which are a placeholder. Also, yes, that is indeed the fun part of the race. You could also say "Every tick, plasmamen will check if they're wearing a specific jumpsuit and helmet and, if not, they will ignite." and say "Wow" equally much.

@BeeSting12 How?

**Comment by lzimann** [comment]
I'll probably close this in 24h, I really don't think it's worth having at all, all they have to offer is the vanity thing which does not make any sense to the game at all, and it being a roundstart species makes even less sense.

**Comment by Xhuis** [comment]
The vanity mechanic is a part of their personality, which is the main thing that makes them unique from the other races. You could say that plasmamen have nothing to offer either, because all they have going for them is a fire gimmick that kills them without the right internals and suit. Although I suspect you didn't actually read the PR body, I did state that anything was subject to change; roundstart is what I *want* but I'd be happy for them to get in even without that.

**Comment by jughu** [comment]
my comment was not directed to the pr but uranium

**Comment by Iamgoofball** [comment]
Powergamers will play this because you get a 10% damage reduction to the most common damage and you can greytide easier with the shock reduction

**Comment by AnturK** [comment]
I don't get it.

**Comment by Xhuis** [comment]
@Iamgoofball The speed reduction seems like it'd make it unappealing to powergamers.
@AnturK Don't get what?

**Comment by BeeSting12** [comment]
@Xhuis 
two words: tree huggers

**Comment by Xhuis** [comment]
I agree, preventing climate change and recycling are fetish content.

**Comment by AnturK** [comment]
I mean the theme seems absolutely random?

**Comment by BeeSting12** [comment]
I'm glad you understand

**Comment by Xhuis** [comment]
The fanciness comes from both their inspiration (a race from Pyre) and the ideas I had with characterization. Given that they were trees (large, tall, could be gorgeous and the like), the fluff is that if a tree could walk, talk, and wear fancy hats, that it would probably be incredibly vain because they're used to being comparatively invincible and the largest things in the forest.

**Comment by BeeSting12** [comment]
can lizards climb them

**Comment by Xhuis** [comment]
Um, unless someone coded it behind my back, nope

**Comment by Qustinnus** [comment]
i think this is what a good race PR looks like, it has an interesting gimmick and isn't just a meme / furry race.

can't say i 100% agree with the combat mods but eh those can always be changed later.

**Comment by Dorsisdwarf** [comment]
I think the concept is hilarious and better than any of the other races so I support this PR.

**Comment by uraniummeltdown** [comment]
Lets be real here, a gimmick based on clothing cannot work. People don't really change clothing unless wearing a meme/funny item or wearing a department's clothes for impersonation or job changes

**Comment by Xhuis** [comment]
[citation needed]

**Comment by Pascal125** [comment]
I change my clothing all the time so speak for yourself, pal.

Either way they could like. Just not play the race if they don't feel like clicking a few sprites to change clothes.

**Comment by BeeSting12** [comment]
will they be valid

**Comment by Xhuis** [comment]
Up to admins, although I imagine they'll probably end up dying more often anyway if they're RPed as being posh and vain divas like they're coded as

**Comment by ike709** [comment]
Porting this to FTL13 if TG closes it.

**Comment by Iamgoofball** [comment]
@xhuis movespeed stopped being an issue when it was declared griefing for security to crack down on drug production

**Comment by Qustinnus** [comment]
ike if you port it make it use m o o d i e s

**Comment by jughu** [comment]
>clown suit humilating

**Comment by DoctoreMad** [comment]
"They're all very vain so they all have to wear nice clothes"
No thanks. Seriously, **no**. I appreciate that you're trying to find a good mechanic but not like this, you're basically forcing a specific personality into a whole race.

**Comment by Qustinnus** [comment]
"they all light on fire so they have to wear plasmasuits and breathe plasma"
No thanks. Seriously, no. I appreciate that you're trying to find a good mechanic but not like this, you're basically forcing a specific trait into a whole race.

**Comment by DoctoreMad** [comment]
There's a very big difference on "This race needs X to live" and "Everyone in this race follows this specific philosophy of thought"

**Comment by Qustinnus** [comment]
it's a gameplay mechanic. you're not forced to roleplay being vain. you are however biologically in need of wearing fancy clothes.

**Comment by Slignerd** [comment]
> you're basically forcing a specific personality into a whole race.

Hmn, I'd say that this a fair point actually.

While it's fine for a race to have some sort of dominant culture, individuals will still differ within said culture.

I'm not sure how it applies to this race specifically, but I guess it does somewhat!

**Comment by DoctoreMad** [comment]
>you are however biologically in need of wearing fancy clothes.

That's _almost_ okay so I'll just set this little thumble on your road:
Plasmamen need plasma because their bodies being lit on fire on contact with oxygen.
Saps need fancy clothes because

**Comment by Xhuis** [comment]
There's a thousand different fluff explanations that can be provided for it. Here are some:
* NT only hires the uptight ones because they're either good fodder for stress relief (being aliens they have to work cheaply) or are too prideful to not be good at their jobs
* The only non-xenophobic sect of saps who were interested in cooperating with other races only did so because they viewed themselves as superior
* Saps have an innate tendency to be vain like humans tend towards conflict

It's a good point, although I'm unsure how to address it without outright removing the interesting part of the race.

**Comment by Qustinnus** [comment]
it's almost like this entire game is based on psuedo science and anything can be explained by weird shit just happening. (like everything related to plasma and bluespace stuff)

**Comment by Iamgoofball** [comment]
@DoctoreMad someone makes a spess alien that's got a unique gimmick and you hate it? come on m8

**Comment by Iamgoofball** [comment]
s

**Comment by DoctoreMad** [comment]
This is the ugliest form of "psuedo" science I have ever seen. At least Xhuis tried to explain it in his last post, instead of this ugly "I don't know how to make sense of this so I just won't" 
I should've just told you to stay on the FTL/yogstation gits, but alas, here I am.

@Iamgoofball Says the guy that shitposted on the dullahan PR.
Wait are you trying to "/s" me.

**Comment by The-clown-known-as-Pepe** [comment]
@DoctoreMad dullahans are fantasy, not aliens you dummy

**Comment by Iamgoofball** [comment]
are you high

headless horsemen guys are also a unique mechanic

**Comment by Qustinnus** [comment]
because Xhuis already posted that, there's no point in me repeating what he said. I just added that your argument of "it has no reason to be like this" is shit.

Also telling me to stay on the yog git would be dumb because I'm here more than on the yog git, and I actually make PRs for TG. I might as well tell you to stay off the tg git instead because you don't code.

**Comment by DoctoreMad** [comment]
>I just added that your argument of "it has no reason to be like this" is shit.

Is there a _single_ feature in SS13 that doesn't try to explain itself, regardless of how poor the explanation itself is?

>I might as well tell you to stay off the tg git instead because you don't code.

☭ HAIL CODERBUS ☭

**Comment by Qustinnus** [comment]
the feature is explained though so your point is moot

**Comment by Slignerd** [comment]
> it's almost like this entire game is based on psuedo science and anything can be explained by weird shit just happening. (like everything related to plasma and bluespace stuff)

That's a somewhat poor argument tbh. Some stuff can be handwaved, but that doesn't mean you should throw all logic whatsoever out of the window.

**Comment by kilkun** [comment]
this is good and cool with a fun unique gimmick and the idiots opposed to a non-furry race can fuck off

**Comment by lzimann** [comment]
Oh boy, that's a lot of talking for something that won't get merged

**Comment by DoctoreMad** [comment]
Don't you hate firing on your allies? But you gotta ask what you gotta ask.

>for something that won't get merged

On whose authority are you speaking?

**Comment by Shihrt** [comment]
oh no it's time for daddy leoz to powertrip

**Comment by Iamgoofball** [comment]
@lzimann if we rename it to Remie will you merge it?

**Comment by ike709** [comment]
Since lzimann is going to pull an oranges and close the PR due to personal dislike once it hits 24 hours, you can always open this PR on FTL13 and it will be merged @Xhuis . Or I'll just port it if you don't.

**Comment by Xhuis** [comment]
Regardless of what ends up happening, I do still have some work I'm doing to push to this, with stuff like random names and more distinct appearances.

**Comment by uraniummeltdown** [comment]
@Xhuis I don't want to discourage you from coding because you're a good coder who's added great things and we lost Remie and coiax already

But these just aren't fleshed out enough, it reminds me of that Vat Human race PR that got closed, its just too shortsighted

**Comment by Iamgoofball** [comment]
these are pretty fleshed out compared to klingoffs, space elves, and cattebeest

**Comment by Xhuis** [comment]
How do you mean?

**Comment by Iamgoofball** [comment]
same level of fleshed out as dullahans too

dullahans only have one gimmick as well

**Comment by The-clown-known-as-Pepe** [comment]
@Iamgoofball >one gimmick

Stats:

Slower movement than default humans (around dufflebag speed by default; this is the big one)
60% less shock damage due to their planty flesh
10% less brute damage due to bark
10% more burn damage due to being plants
Heat up 50% faster due to the above
Dislikes meat and dairy, likes fruit, veggies, and grains
Uses E-Z Nutrient for blood
Uses canopies instead of hairstyles

**Comment by DoctoreMad** [comment]
@Iamgoofball Klingoffs and space elves were, well, what the hell did they even have anything?
Anyways the comment is completely out of the blue.

>dullahans only have one gimmick as well

If you do one gimmick well that's enough.

@The-clown-known-as-Pepe 
Stat changes technically are a gimmick but do you REALLY want to count those? Remember that every different golem we have is also a race.

**Comment by Iamgoofball** [comment]
1. this is already a coded feature (golems use this)
2. see above (we've had species DR support for ages)
3. see above (we've had species DR support for ages)
4. see above (we've had species DR support for ages)
5. see above (we've had species temperature sensitivity for ages, iirc lizards use it)
6. see above (we've had food preference code for quite some time now)
7. see above (slimes having slime jelly for blood)
8. it's just renamed hairstyles

they only have 1 unique gimmick, which is good enough

**Comment by uraniummeltdown** [comment]
Basically if you found some WHOA MAN NICE!!! sprites or maybe gave them some cool kind of innate ability it would be fine

Paradise has antpeople who can use nutrient to glow in the dark and produce light
They have ayy lmaos that have special telepathy
Cult and Bluespace golems also have neat abilities

The fanciness thing only is visible to the user and it hardly matters

**Comment by Iamgoofball** [comment]
@uraniummeltdown all those paradise bits were stolen from bay/vg

the wolfkin doggebeest is their only original(if you can even fucking call it that) race

**Comment by Xhuis** [comment]
I'd rather not have sprites be high-visibility or over-the-top since they're focused on the game and generally not the player. As for unique abilities, I'm open to suggestions, but /tg/ takes some prodding because anything that isn't a human with differently-colored skin causes an eruption of vitriol.

**Comment by DoctoreMad** [comment]
Oh great, is the Paradise community here?

**Comment by Xhuis** [comment]
Kor really was right, Xhuis PRs accumulate dozens of comments in a very short timespan.

**Comment by uraniummeltdown** [comment]
@Iamgoofball greys are from vg but antpeople are original
my point is give the saps some cool ability or unique thing, nobody notices stat changes

**Comment by Iamgoofball** [comment]
@DoctoreMad are you high

**Comment by Iamgoofball** [comment]
no, the resomi/glowy ant people are from bay and Polaris, made by Zuhayr

**Comment by DoctoreMad** [comment]
No, this time I am very down to earth. Whenever someone says Paradise is original all my sensor flags flare.

**Comment by uraniummeltdown** [comment]
dude kidan was added to paradise in 2013 thats before Polaris was even made

**Comment by uraniummeltdown** [comment]
holy shit goof is right, even kidan is from bay

**Comment by ike709** [comment]
paradise didn't even exist in 2013

**Comment by Arianya** [comment]
I like the idea (also being a fan of Pyre), but do agree that the fanciness thing is... a bit too subtle? I guess it'd be notable because you'd regularly see Saps dressed in the fanciest gear they can get their hands on, but equally getting hungry tends not to be a barrier. The slowdown is more of a barrier but its also not neccesarily "unique".

Anyway this is all moot unless another maintainer or the headcoder step in since Leo gets upset when someone tries to implement new races.

(Still waiting to hear what an interesting race is by your definition btw lzi)

**Comment by kilkun** [comment]
argonians

**Comment by DoctoreMad** [comment]
It doesn't matter anyways because if we had enough creativity to make a Star Control-tier race we would have frumpled in the amount of shitcode this game has.

**Comment by ExcessiveUseOfCobblestone** [comment]
I think I could be ok with this if the sprites were also PRd or at least not set to roundstart until they are given unique sprites.

**Comment by Xhuis** [comment]
There's some spritework (not new stuff but modified and more fitting) coming within an hour or so that splits their canopy's color from their bark's color and adds the plant hairstyles as canopies. New canopies are/would be easy to add since they function just like hairstyles.

**Comment by Xhuis** [comment]
With the next batch of changes, Saps can look much more distinct from both one another and from the podpeople race they're based off of:

**Comment by ShadowDeath6** [comment]
not a big fan

make fanciness actually mean something; making it ignorable just makes it that much less of a feature. I'd prefer it do something other than starve you because starving is the least interesting mechanic 

plasmamen are ok in my eyes because it's literally "if you take off this suit you die almost 100% of the time in any situation" and "breathe plasma or you die", it's not "slowly get hungry if you aren't dressed in x and y"

damage resistances are The Fuck Shit and should not ever be used as features

**Comment by ShadowDeath6** [comment]
I can't edit on mobile: by damage resistances not being features I meant that they shouldn't be used as "cool feature that makes the race stand out", not that they shouldn't be implemented at all

**Comment by kilkun** [comment]
rip out all mob code and replace it with a system that emulates how dwarf fortress handles mobs. bam done. solved. promote me to head ideas guy status any time my dudes.

**Comment by Xhuis** [comment]
The typecache system of fanciness has been replaced with a new var on `/obj/item/clothing`, called `fanciness`. There are also defines such as `FANCY_DISTASTEFUL` and `FANCY_LAVISH` to make designating fancy items easier.

**Comment by rustledjimm** [comment]
@Izimann since this is just a pr to see how people feel about the idea and xhuis was going to close it after some discussion so he could work more on it anyway I'm curious why you're so set on closing it yourself? Just let Xhuis close it and work on it considering it has garnered some good reactions even before it's been properly fleshed out.

**Comment by Xhuis** [comment]
I didn't plan on closing it myself, but I rather expected backlash specifically from leo because he has a history of disliking PRs I make in particular. Since I PRed them they have far more polish and are actually different from podpeople, so I'd be alright with its merging even if it didn't end up as roundstart.

**Comment by Bawhoppen** [comment]
The balance changes are probably a little too strong, and maybe would be better if turned down. 
I think ideally, we'd also need more clothing for this to work correctly (which we honestly need regardless of this or not, since we have a current lack of clothing choices)

In general, I feel the whole thing could use another layer of polish, though I honestly can't manage to determine what that should be.

**Comment by DoctoreMad** [comment]
Should I apologize for being such a sap?

**Comment by Xhuis** [comment]
😎 👉 👉

File: strings/names/sap_title.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+
+
+I
+II
+III
+Esq.
+Sr.
+Jr.
```
Comment (Hatfish) [inline strings/names/sap_title.txt]: Isn't it against our current name rules to have titles like this?

Comment (rustledjimm) [reply strings/names/sap_title.txt]: I mean the other day I saw a Captain going around with the name X Of Syndicate. So no.

Comment (DoctoreMad) [reply strings/names/sap_title.txt]: You can argue that Saps would put Sr. And Jr. in their actual names.

Comment (DoctoreMad) [reply strings/names/sap_title.txt]: I SERIOUSLY THINK IT SHOULDN'T BE HERE THOUGH, FOR REFERENCE

**Comment by ChangelingRain** [comment]
Fanciness should be tracked better than just 'this item is fancy', since a top hat goes TERRIBLY with work gloves like the botanist's, and similarly, a crown just doesn't work with a suit.

Similarly, wearing only part of an outfit, like the xeno costume, should be much worse than actually wearing the entire thing, since fanciness isn't merely a matter of how good each part of an outfit is, *but how well the entire outfit works together*.

**Comment by Qustinnus** [comment]
from atmos sim to fashion sim.
boy what an advancement :^)

i agree with Joan though, maybe make groups of items that fit well together as one piece but some pieces just clash with others? flags could work.

**Comment by Xhuis** [comment]
The way to implement outfits that change fanciness depending on what's worn seems like it'd be fairly easy to do by just using datums called `/datum/sap_outfit` or somesuch. I don't think there's otherwise a good way to check how pieces work in tandem.

**Comment by duncathan** [comment]
i am opposed to lzi here and i would love to see this reach a state where we can have these merged. sounds like a load of fun to play as tbh

**Comment by ChangelingRain** [comment]
In general I'd prefer outfit datums for checking, rather than a var on all clothing used only for the race. Much more modular.

**Comment by Iamgoofball** [comment]
that locks everyone into set outfits

**Comment by DoctoreMad** [comment]
Outfit spangalooza of whatever Saps can find sounds like a good idea for me.

**Comment by KorPhaeron** [comment]
@duncathan Shouldn't they at least not be lifted wholesale, name and all, from another game when we already have a plant people race? People are touchy enough about the extra races and roundstart species that this isn't spectacular enough for me to want to fight through that

Also @Armhulen and @ausops would you vote to enable these

**Comment by ChangelingRain** [comment]
>that locks everyone into set outfits

only if coded badly.

**Comment by Iamgoofball** [comment]
yes, but this is an Xhuis PR, we all know you aren't going to help him at all and are just coming up with this as an excuse to close it in a few weeks

**Comment by Xhuis** [comment]
@KorPhaeron I stuck with the name Saps only because it has character and rolls off the tongue; if you'd like I'm fine with changing the name.

I was actually thinking of *replacing* podpeople with this if they didn't become roundstart, or at least a variant of these that had the same functionality as the old, since the sprites are more modular and support different hairstyles/canopies being added like lizards have horns and the like. I've always viewed podpeople as purposefully very similar to humans, given how they're made, so adding a unique race with a similar plant theme doesn't seem too bad to me either way

**Comment by Xhuis** [comment]
As for outfits, I'd be willing to do it, but I also kind of intentionally left that function out. Saps try to be fancy, but being that they're tree-people and thus not exactly quick to learn new cultures, they view "random clothing that looks good" = "good outfit!" and would probably become extremely indignant at someone trying to suggest that their mishmash of clothing doesn't just tie off their canopy *so* well.

**Comment by fludd12** [comment]
I would've picked another mechanic entirely if you were going for something Pyre-like, though that guy *was* a big enough asshole to bring down the whole race. It just seems kinda... out of place? It definitely doesn't fit with the whole idea of being a plant, unless you consider flowers as being intentionally flaunty.

Regardless, this *is* your idea, and I'm not one to shit all over an idea just because I'm iffy on a mechanic I haven't even tried out yet. I'm interested to see where this goes in the future!

**Comment by optimumtact** [comment]
into the trash heap most likely

**Comment by Doppeltongue** [comment]
As a replacement to pod people, it doesn't feel planty enough. Especially when you consider that you'll have humans get replica podded/plant mutation toxin'd and then suddenly need to wear fancy clothes because... reasons? It'll also require upkeep every time new apparel is added, and "fanciness" is a really subjective metric.  I'm pretty sure it's also missing the plant and vine flags.

I like the idea of a new plant-race, but the whole dress up thing makes no sense for an in-game mechanic. It's great flavor for RPing, but not for playing. Instead, how about the race gets negative effects from being covered in blood? Give them minor effects from bloody shoes, and worse effects for bloody hands/clothes. Maybe they're more badly affected by grossness from disliked foods, too. This plays into their vanity without getting too arbitrary about what classifies as good fashion.

**Comment by DoctoreMad** [comment]
Yeah, having a race that people hardly liked on the very PR they were added replacing a older and more well known race is just too much.

**Comment by lzimann** [comment]
ok let's do this and never even think about a fancy mechanic again!

**Comment by DoctoreMad** [comment]
Were you actually waiting for the time 23 hours would turn into "a day"

**Comment by Pascal125** [comment]
Yes, he was. Also. I don't even remember commenting on this.

**Comment by Iamgoofball** [comment]
He has a script running

@KorPhaeron can you reopen this, this is a good pr

**Comment by lzimann** [comment]
No script, just good timing this time actually

**Comment by DoctoreMad** [comment]
Not all gimmicks are good gimmicks, Goofball.
You could make a race that only has one hand in turn of having telekinesis by default but I doubt many people would like that.

**Comment by Xhuis** [comment]
This is about what I expected, so I'm not entirely surprised. If any maintainers want to see this worked on further, reopen the PR or at least let me know so I can commit to the branch, but otherwise this is dead. vOv

**Comment by Doppeltongue** [comment]
@Xhuis I think it was a good idea to try and make a heavier RP race, but the fanciness mechanic would've been too limiting. Instead of people putting together interesting outfits, the arbitrary fanciness levels of clothing would force players to never wear certain clothes and to just code-dive to find a set of acceptable clothing and then nothing. There's no real interaction with fanciness after that, either you've found an acceptable set or you get hungry... like normal crew. RP should encourage interactions, and clothing isn't interactive, it's just a binary check.

I think something to do with cleanliness is a better choice for vanity, since it would encourage people to avoid contaminated areas and wash themselves regularly, as well as clean the station.

**Pull request closed without merging**